packages/widget/src/components/TokenList/TokenList.tsx

```diff
@@ -12,69 +14,73 @@ import type { TokenListProps } from './types.js'
 import { useTokenSelect } from './useTokenSelect.js'
 import { VirtualizedTokenList } from './VirtualizedTokenList.js'
 
-export const TokenList: FC<TokenListProps> = memo(
-  ({ formType, parentRef, height, onClick }) => {
-    const emitter = useWidgetEvents()
+export const TokenList: FC<TokenListProps> = memo(({ formType, headerRef }) => {
+  const navigateBack = useNavigateBack()
+  const listParentRef = useRef<HTMLUListElement | null>(null)
+  const { listHeight } = useListHeight({
+    listParentRef,
+    headerRef,
+  })
```

Comment:
I'm not sure why the height calculation was inside WrappedTokenList, but have you checked that it doesn't affect performance if we move it here? 👀 Also, just generally curious if that was applied in relation to changing the routing to tanstack.